cclib/parser/data.py

```diff
@@ -169,8 +193,13 @@ class ccData:
        "scfenergies":      Attribute(numpy.ndarray,    'scf energies',                'optimization:scf'),
        "scftargets":       Attribute(numpy.ndarray,    'targets',                     'optimization:scf'),
        "scfvalues":        Attribute(list,             'values',                      'optimization:scf'),
+       "solvation":        Attribute(str,              'solvation',                   'metadata'),
+       "symmno":           Attribute(int,              'symmetry number',             'properties:rotational'),
+       "td_energy":        Attribute(list,             'TD energy',                   'optimization:TD'),

```

Comment:
This option was designed for GoodVibes, we wanted the program to automatically use the TD energy in TD energy profiles. In this case, GoodVibes checks if the attribute "td_energy" exists to determine whether to use this energy or the normal scf energy. What would you suggest? (I'm not familiar with the "opt_state" data)

Reply:
Ah ok I think I see.

Excited state energies are available in `etenergies` which should work fine for most situations. However, at the moment we only parse the 'final' energies, which for following along with an optimisation is no good, because you'll want to know the energies at each excited state step. I think it's best we open a new issue for this, because changing `etenergies` is non backwards compatible.

For the final energy however you are already covered (ie, the equivalent of the new `td_energy_fin` you added). `metadata['opt_state']` indicates which excited state is being optimised, so `etenergies[metadata['opt_state']]` will give you the final energy of the optimised excited state. If you then add on the total system energy in `scfenergies[-1]` you will get the value you are looking for.

Reply:
https://github.com/cclib/cclib/issues/1273

Let's remove these attributes from this PR for now

Reply:
sounds good